native/objcexport-header-generator/testData/headers/enumClassWithObjCEnum/Foo.kt

```diff
@@ -0,0 +1,15 @@
+import kotlin.native.ObjCEnum
+import kotlin.native.ObjCName
+import kotlin.experimental.ExperimentalObjCEnum
+import kotlin.experimental.ExperimentalObjCName
+
+@file:OptIn(ExperimentalObjCEnum::class)
+
+@ObjCEnum
+enum class Foo {
+    ALPHA_BETA,
+    ALPHA,
+    COPY,
+    @ObjCName("fooBarObjC") BAR_FOO,
+    @ObjCName(name = "barFooObjC", swiftName = "barFooSwift") BAR_FOO,
```

Comment:
Does this `enum class` have two enum entries with the same name? How does it work?